package.json

```
@@ -25,11 +25,10 @@
   "dependencies": {
     "file-type": "^3.3.0",
     "html-comment-regex": "^1.1.2",
+    "@aws-sdk/lib-storage": "^3.46.0",
```

These should be sorted, since otherwise Npm will sort them the next time something is installed which will generate extra diff churn...

```suggestion
    "@aws-sdk/lib-storage": "^3.46.0",
    "file-type": "^3.3.0",
    "html-comment-regex": "^1.1.2",
```